app/components/add-cron-job.js

```diff
@@ -0,0 +1,59 @@
+import Ember from 'ember';
+import config from 'travis/config/environment';
+
+export default Ember.Component.extend({
+  classNames: ['form--cron'],
+  store: Ember.inject.service(),
+
+  reset() {
+    return this.setProperties({
+      branch: null,
+      interval: null,
+      disable_by_build: null
+    });
+  },
+
+  actions: {
+    save() {
+      var cron, self, store, repo_id, branch;
+
+      if (this.get('isSaving')) {
+        return;
+      }
+
+      self = this;
+      store = this.get('store');
+      repo_id = this.get('branches').toArray()[0].get('repoId');
+      branch = this.get('selectedBranch') ? this.get('selectedBranch') : this.get('branches').toArray()[0];
+
+      store.filter('cron', {
+        repository_id: repo_id
+      }, function(c) {
+        return c.get('branch.repoId') === repo_id && c.get('branch.name') === branch.get('name');
+      }).then(function(existing_crons){
+        if(existing_crons.toArray()[0]){
+          store.unloadRecord(existing_crons.toArray()[0]);
```

Comment:
Only one cronjob can exist per branch. If you add a new one, the API will delete the old one for this branch (if existing). Here we unload the old cron (which will be deleted in the API), so the autorefesh works.